Add chain transition FAQ page (#18)
kaiachain/kaia-docs: docs/misc/faq-chain-transition.md

- [What support will Kaia Foundation provide for the migration of Klaytn and Finschia apps?](#what-support-will-kaia-foundation-provide-for-the-migration-of-klaytn-and-finschia-apps-)
- [Will Kaia Foundation continue to provide code audit grants as Klaytn Foundation did?](#will-kaia-foundation-continue-to-provide-code-audit-grants-as-klaytn-foundation-did-)

## As an RPC node provider, what changes do I need to make to my infrastructure to support the transition from Klaytn to Kaia, and will there be any breaking changes in the RPC APIs? <a id="as-an-rpc-node-provider-what-changes-do-i-need-to-make-to-my-infrastructure-to-support-the-transition-from-klaytn-to-kaia-and-will-there-be-any-breaking-changes-in-the-rpc-apis-"></a>

Most RPC APIs will remain the same on the Kaia chain, with the exception of namespace changes from klay_ to kaia_. However, klay_ will still be available for backward compatibility.

To support the transition from Klaytn to Kaia, RPC node providers will need to upgrade their binary to the Kaia version once it is released. No immediate action is required before the upgrade.

## Will the Network ID change? <a id="will-the-network-id-change-"></a>

> No, the Klaytn Network ID will remain the same. The original URL will be active for three months to ensure a smooth transition.

Reviewer: Please say like "Network ID (Chain ID)" because Chain ID is also widely used.

Also the URL is not directly related to Network ID I guess..?

Reviewer: It looks nicer to merge the two questions!

Still, it's unclear what "The original URL" means in this context. A _chain_ doesn't have its URL. Does it mean the klaytn websites? public endpoints? or any URLs and domains maintained by the Klaytn Foundation? I think this last line needs examples of those original URLs.

e.g.
... is now referred to as "Mainnet" or "Kaia Mainnet." The original URLs related to Klaytn such as docs, website, public endpoints, and package download links will remain for three months to ensure a smooth transition.

Author: @blukat29 I've made an additional commit to apply feedbacks. Please check out the changes.

Reviewer: Looks good. Thanks

## Will Kaia be on the Klaytn mainnet or a different mainnet? <a id="will-kaia-be-on-the-klaytn-mainnet-or-a-different-mainnet-"></a>

Kaia will continue to be on the Klaytn mainnet.

## Will the current Klaytn mainnet continue to work after the rebrand? <a id="will-the-current-klaytn-mainnet-continue-to-work-after-the-rebrand-"></a>

Yes, the mainnet's operation will remain unaffected. The transition to Kaia primarily involves a name change and technical upgrades occurring behind the scenes.

## When will the KAIA market open? <a id="when-will-the-kaia-market-open-"></a>

The KAIA market is scheduled to open on June 27th. The exact block number will be shared closer to the date.

## Will Klaytn provide a swap or migration website for new KAIA tokens? <a id="will-klaytn-provide-a-swap-or-migration-website-for-new-kaia-tokens-"></a>

No action is required from users. KLAY tokens will be automatically renamed to KAIA tokens.

## Will there be any delay in receiving KAIA tokens? <a id="will-there-be-any-delay-in-receiving-kaia-tokens-"></a>

> Due to the Timelock function, token swap finalization will be delayed by at least 30 minutes. Additional delays, such as relayer and RPC delays, may occur but are not significant.

Reviewer: ```suggestion
The token swap will be delayed by at least 30 minutes Timelock for safety. Relayers and RPC nodes may incur insignificant additional delays.
```
